src/macros.js

```diff
@@ -289,6 +289,11 @@ defineMacro("\\varPhi", "\\mathit{\\Phi}");
 defineMacro("\\varPsi", "\\mathit{\\Psi}");
 defineMacro("\\varOmega", "\\mathit{\\Omega}");
 
+// \renewcommand{\colon}{\nobreak\mskip2mu\mathpunct{}\nonscript
+// \mkern-\thinmuskip{:}\mskip6muplus1mu\relax}
+defineMacro("\\colon", "{\\mkern2mu\\mathpunct{}" +
+    "\\mathchoice{\\mkern-3mu}{\\mkern-3mu}{}{}{:}\\mkern7mu}");
+
```

Comment:
`7mu` should be `6mu`. (`plus` does not add, it states allowable growth when stretching.)

I think you should also add `\nobreak` at the beginning (which prevents a line break before the macro), and I also feel like the outer `{}` should be removed to match LaTeX... Could you test it in this form to make sure there are no bad breaks?

As a minor thing, I'd change `\mkern` to `\mskip` to match the macro definition as closely as possible, but they are of course identical at this point.